llvm/include/llvm/Analysis/TargetLibraryInfo.h

```diff
@@ -156,6 +156,10 @@ class TargetLibraryInfoImpl {
   /// FDecl is assumed to have a parent Module when using this function.
   bool getLibFunc(const Function &FDecl, LibFunc &F) const;
 
+  /// Searches for a function name using an Instruction \p Opcode.
+  /// Currently, only the frem instruction is supported.
+  bool getLibFunc(unsigned int Opcode, Type *ScalarTy, LibFunc &F) const;
```

Comment:
I'd just use `Ty` instead of `ScalarTy` because the interface doesn't care what the type is.